.aegir.js

```diff
@@ -15,8 +15,14 @@ module.exports = {
   },
   hooks: {
     browser: {
-      pre: server.start.bind(server),
-      post: server.stop.bind(server)
+      pre: (callback) => {
+        server.start()
+          .then(() => callback(), callback)
+      },
+      post: (callback) => {
+        server.stop()
+          .then(() => callback(), callback)
+      }
     }
```

Comment:
can you change this to just return the promise, aegir with support for hooks promise is release

Reply:
Done!